src/code/audio_load.c

```diff
@@ -790,83 +791,145 @@ AudioTable* AudioLoad_GetLoadTable(s32 tableType) {
     return table;
 }
 
-void AudioLoad_RelocateFont(s32 fontId, SoundFontData* mem, RelocInfo* relocInfo) {
-    u32 reloc;
-    u32 reloc2;
+/**
+ * Read and extract information from soundFont binary loaded into ram.
+ * Also relocate offsets into pointers within this loaded soundFont
+ *
+ * @param fontId index of font being processed
+ * @param fontData ram address of raw soundfont binary loaded into cache
+ * @param relocInfo information on the sampleBank containing raw audio samples
+ */
+void AudioLoad_RelocateFont(s32 fontId, SoundFontData* fontData, SampleBankRelocInfo* relocInfo) {
```

Comment:
```suggestion
void AudioLoad_RelocateFont(s32 fontId, SoundFontData* fontData, SampleBankRelocInfo* sampleBankRelocInfo) {
```
yeah it's long but it should be even clear(er) that it has nothing to do with sound font data relocation directly

Reply:
I wonder if `sampleBankReloc` would be clear enough? (And keep `SampleBankRelocInfo` as the struct name)